### PR TITLE
Add tree planting option

### PR DESCRIPTION
**Pull request by loneil**
# Description
https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-1858
Add 'Tree Planting' as an option, keep 'Silviculture too'.

Db migration and FE change.

Also update link references from pathfinder to silver.

## Types of changes

New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by bcgov-nr-csst** [comment]
PR Deployed at: https://comfort-dev.apps.silver.devops.gov.bc.ca/pr-160

File: local-infrastructure/README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Application setup details and more are in the [Developer Guide](# https://bcgov.
 
 ### Quick Run Steps
 
-1. Sign up at the [GETOK Common Service Onboarding page](https://getok.pathfinder.gov.bc.ca/getok/about).
+1. Sign up at the [GETOK Common Service Onboarding page](https://getok.nrs.gov.bc.ca/getok/about).
```
Comment (jujaga) [inline local-infrastructure/README.md:30]: Path is incorrect - should be https://getok.nrs.gov.bc.ca/app/about

Comment (loneil) [reply local-infrastructure/README.md:30]: 👍

**Review by TimCsaky — APPROVED** [review]
All looks good!

**Review by jujaga — APPROVED** [review]

**Pull request merged**